src/commands/info.rs

```diff
@@ -0,0 +1,30 @@
+use color_eyre::{Report, Result};
+use single_instance::SingleInstance;
+
+use crate::summary::{get_farmed_block_count, get_initial_plotting_progress};
+
+pub(crate) async fn info() -> Result<()> {
+    let instance = SingleInstance::new("subspaceFarmer").map_err(Report::msg)?;
+    if !instance.is_single() {
```

Comment:
you are right, it was an error because I thought since the application has started, there are multiple locks acquired. However, only the farmer takes the lock. My bad

Reply:
@jfrank-summit logic was correct. The lock was getting dropped too early, that was the bug. I've solved it